package.json

```diff
@@ -56,6 +56,7 @@
     "@microsoft/api-documenter": "^7.14.1",
     "@microsoft/api-extractor": "^7.19.4",
     "@microsoft/api-extractor-model": "^7.15.3",
+    "@veramo/cli": "4.2.0",
```

Comment:
Why is @veramo/cli needed? I do not see it being used from first glance

Reply:
removed. I was running the veramo command from command line for which I was trying different things.